packages/js/src/core/user-info.ts

```diff
@@ -7,12 +9,12 @@ type Identity = {
 
 export type UserInfoResponse = {
   sub: string;
-  name?: string;
-  username?: string;
-  picture?: string;
-  email?: string;
+  name?: Nullable<string>;
+  username?: Nullable<string>;
+  picture?: Nullable<string>;
+  email?: Nullable<string>;
   email_verified?: boolean;
-  phone_number?: string;
+  phone_number?: Nullable<string>;
   phone_number_verified?: boolean;
   custom_data?: unknown;
   identities?: Record<string, Identity>;
```

Comment:
just double check these two fields. if they won't be `null` maybe we can add some comments?

Reply:
I checked DB and it seems these jsonb data will have `{}` as default value. Comments added, ptal